index.js

```diff
@@ -52,6 +52,23 @@ function toggleSound(){
         timer.innerHTML = "<b>" + counter + "</b>";
     }, 1000);
 
+    // refresh when reload
+    const showAllCards = ()=>{
+        cards.forEach((card, i)=>{
+            card.children[0].src=cardImageSrcs[i]; // Setting image source for flipped front face
+            card.children[0].alt=cardImageSrcs[i].split('/').slice(-1)[0].split('.').slice(0, -1).join('.'); // Setting image file name as alt text for flipped front face
+            card.children[1].style.display="none";
+        })
+        
+        setTimeout(()=>{
+            cards.forEach((card)=>{
+                card.children[0].src="#"; // Removing image src so that it isn't visible through HTML
+                card.children[0].alt="card front face"; // Removing image alt so that it isn't visible through HTML
+                card.children[1].style.display="block";

```

Comment:
this same logic is used in the flip card function. So just remove the Repeatation

Reply:
Okay, I'll create separate function for this